tests/performance-test/run.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+TIMESTAMP=$(date +%Y_%m_%d_%H_%M_%S)
+
+cd results
+exec poetry run locust --users 1 --run-time 30s -H 'https://main.d1ie8muegpw33w.amplifyapp.com' -f ../shared.py,../create_nhsapp_template.py --class-picker $*  --user-name=[username] --password=[password] --html="report_$TIMESTAMP.html" --log-transactions-in-file --logfile="log_$TIMESTAMP.txt"
```

Comment:
i don't think we want hardcoded URLs anywhere

Reply:
Removed the hardcoded URL, it will be now passed when the test will be started.